### ISSUE TITLE
BUG: Debian instructions missing libc6-dev-i386

### ISSUE THREAD

**Issue by mgerdts** [linux-audit/audit-testsuite#71]
On Ubuntu 16.04, the instructions in README.md lead to:

```
cc -g -O0 -Wall -D_GNU_SOURCE -m32 -o conn conn.c
In file included from /usr/include/unistd.h:25:0,
                 from conn.c:1:
/usr/include/features.h:367:25: fatal error: sys/cdefs.h: No such file or directory
compilation terminated.
Makefile:8: recipe for target 'conn' failed
```

Fixed with:

```
diff --git a/README.md b/README.md
index 625a684..c6124c7 100644
--- a/README.md
+++ b/README.md
@@ -53,6 +53,7 @@ please follow the instructions below.
        # apt-get install auditd \
                          build-essential \
                          libc6-i386 \
+                         libc6-dev-i386 \
                          perl-modules \
                          netcat
```

**Comment by pcmoore** [comment]
Thanks @mgerdts.  Would you be willing to submit a proper fix via email or PR as described in [CONTRIBUTING.md](https://github.com/linux-audit/audit-testsuite/blob/master/CONTRIBUTING.md)?

**Comment by pcmoore** [comment]
This was evidently fixed in 2e53d6659a221171b0790df5d7c89cb3c829b005, closing.
